app/controllers/application_controller.rb

```diff
@@ -107,7 +107,7 @@ def api_deprecation_error(exception = nil)
   def smart_proxy_exception(exception = nil)
     process_error(:redirect => :back, :error_msg => exception.message)
     logger.warn "ProxyAPI operation FAILED: #{exception}"
-    logger.debug exception.backtrace.join("\n")
+    logger.warn exception.backtrace.join("\n")
```

Comment:
Adding a message is usually pretty important for context - otherwise you'd have to trace back from the log file to figure out what called it.  All of the instances you've changed in this PR have some message that adds context.